common/formatting/token_partition_tree.cc

```diff
@@ -423,10 +423,11 @@ TokenPartitionTree* MergeLeafIntoNextLeaf(TokenPartitionTree* leaf) {
 //
 class TokenPartitionTreeWrapper {
  public:
-  TokenPartitionTreeWrapper(const TokenPartitionTree& node) : node_(&node) {}
+  TokenPartitionTreeWrapper(const TokenPartitionTree& node)  // NOLINT
```

Comment:
Ok to make this `explicit`

Reply:
This is one of these wrapper types where explicit conversion is desired. Adding an `/* implicit */` comment in front.